cron_tasks.go

```diff
@@ -0,0 +1,27 @@
+package scalingo
+
+import (
+	"gopkg.in/errgo.v1"
+)
+
+type CronTasksService interface {
+	CronTasksAdd(app string, params CronTasks) (*CronTasks, error)
+}
+
+type Job struct {
+	Command string `json:"command"`
+	Size    string `json:"size"`
+}
+
+type CronTasks struct {
+	Jobs []Job `json:"jobs"`
+}
+
+func (c *Client) CronTasksAdd(app string, params CronTasks) (*CronTasks, error) {
```

Comment:
Don't use pointer if you don't have a good reason to use a pointer:

```suggestion
func (c *Client) CronTasksAdd(app string, params CronTasks) (CronTasks, error) {
```